src/components/Layout/Navbar/Navbar.component.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faBars, faUser } from '@fortawesome/free-solid-svg-icons'
```

Comment:
I recommend to import the fontawesome icons using the `Icon Library` approach, this way your bundle get smaller. This is what fontawesome suggests:

https://fontawesome.com/how-to-use/javascript-api/setup/importing-icons

Once imported, you can use `FontAwesomeIcon` with the prop `icon='name'` to actually use it.